src/vs/workbench/contrib/chat/browser/chatContentParts/chatSuggestNextWidget.ts

```diff
@@ -0,0 +1,216 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as dom from '../../../../../base/browser/dom.js';
+import { Button, ButtonWithIcon } from '../../../../../base/browser/ui/button/button.js';
+import { Codicon } from '../../../../../base/common/codicons.js';
+import { Emitter, Event } from '../../../../../base/common/event.js';
+import { Disposable } from '../../../../../base/common/lifecycle.js';
+import { observableValue, autorun } from '../../../../../base/common/observable.js';
+import { localize } from '../../../../../nls.js';
+import { IChatMode } from '../../common/chatModes.js';
+
+export interface INextPromptSelection {
+	readonly promptId: string;
+	readonly description: string;
+}
+
+export class ChatSuggestNextWidget extends Disposable {
+	public readonly domNode: HTMLElement;
+
+	private readonly _onDidChangeHeight = this._register(new Emitter<void>());
+	public readonly onDidChangeHeight: Event<void> = this._onDidChangeHeight.event;
+
+	private readonly _onDidSelectPrompt = this._register(new Emitter<INextPromptSelection>());
+	public readonly onDidSelectPrompt: Event<INextPromptSelection> = this._onDidSelectPrompt.event;
+
+	private readonly _isExpanded = observableValue<boolean>(this, true);
+	private headerButton!: ButtonWithIcon;
+	private actionButton: Button | undefined;
+	private actionContainer!: HTMLElement;
+	private contentContainer!: HTMLElement;
+	private promptListContainer!: HTMLElement;
+	private _currentMode: IChatMode | undefined;
+	private _firstPromptId: string | undefined;
+	private _firstPromptDescription: string | undefined;
+
+	constructor() {
+		super();
+		this.domNode = this.createSuggestNextWidget();
+	}
+
+	public get height(): number {
+		return this.domNode.style.display === 'none' ? 0 : this.domNode.offsetHeight;
+	}
+
+	public getCurrentMode(): IChatMode | undefined {
+		return this._currentMode;
+	}
+
+	/**
+	 * Updates the ARIA label for the widget to announce expanded/collapsed state
+	 */
+	private updateAriaLabel(element: HTMLElement, modeLabel: string, expanded: boolean): void {
+		const label = localize('chat.suggestNext.title', "Continue with {0}?", modeLabel);
```

Comment:
This label feels a bit confusing. Do we even need it?
If I wanted to act on one of the items in the list, I’d just click it. Ignoring those options already implies I’m continuing in the current mode.
“Continue with RubberDuck” reads like I need to click it just to keep chatting, which isn’t intuitive.

<img width="869" height="151" alt="Image" src="https://github.com/user-attachments/assets/ce4cd55c-59ef-4326-a2f0-490e17d1ca01" />

Reply:
cc: @unthinkmedia

Reply:
Fixed with latest UI :"Procseed with {mode}"